ProjectSettings/TagManager.asset

```diff
@@ -3,16 +3,17 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags: []
+  tags:
+  - PickUp
   layers:
   - Default
   - TransparentFX
   - Ignore Raycast
   - 
   - Water
   - UI
-  - 
-  - 
+  - PickUp
+  - Interactable
```

Comment:
You do check for tags on playerpickup enter/exit and have layers. Couldn't layers lead to not needing to check for tags? Also controlling the collision matrix for pickups/interactables could be an improvement, right?